rdf-abac-core/src/main/java/io/telicent/jena/abac/labels/LabelsStoreRocksDB.java

```diff
@@ -297,7 +324,7 @@ private DBOptions configureRocksDBOptions() {
             if ( !openFlag.compareAndSet(true, false) ) {
                 throw new RuntimeException("Race condition during failing " + CREATE_MESSAGE);
             }
-            LOG.error("Unable to open/create RocksDB label store: " + dbPath, e);
+            LOG.error("Unable to open/create RocksDB label store: {}", dbPath, e);
```

Comment:
SLF4J is quite clever and treats throwables differently to other parameters so that you don't need a {} for it and it will always log the exception message and stack trace. 

The existing Labels class has a number of examples doing the same (see [here](https://github.com/Telicent-io/rdf-abac/blob/db6d26cdc0083bfb7c932f822fa677b29c1c21c8/rdf-abac-core/src/main/java/io/telicent/jena/abac/labels/Labels.java#L115
)  for example).

I could probably route around in the online manuals for a reference but I hope you'll take it on trust = )

Reply:
That's fine, I just had vague recollections of getting into situations where stuff wasn't logged for some reason but that might have been using a different API/logging provider

Reply:
Yeah....there may be an implementation issue that behaves differently..... that's a thought that'll fester.